### ISSUE TITLE
Installation: Failed to enable the Mitsuba Blender Add-on

### ISSUE THREAD

**Issue by OmegaZhou** [mitsuba-renderer/mitsuba-blender#42]
**Failed to enable the Mitsuba Blender Add-on**
Failed to enable the Mitsuba Blender Add-on

**Bug Description**
I use pip to install the Mitsuba and specify version 3.0.1

When I enable the Mitsuba Blender Add-on, the report shows that `AttributeError: module 'mitsuba' has no attribute 'set_variant'`

And when I `import mitsuba` and call the same code in cmd and, it works well

**To Reproduce**
1. Download the latest release from the [release section](https://github.com/mitsuba-renderer/mitsuba-blender/releases).
2. In Blender, go to Edit -> Preferences -> Add-ons -> Install.
3. Select the downloaded ZIP archive.
4. Find the add-on using the search bar and enable it.

**Expected behavior**
```
Traceback (most recent call last):
  File "C:\Program Files\Blender Foundation\Blender 3.3\3.3\scripts\modules\addon_utils.py", line 369, in enable
    mod.register()
  File "C:\Users\dgsd\AppData\Roaming\Blender Foundation\Blender\3.3\scripts\addons\mitsuba-blender\__init__.py", line 260, in register
    if try_register_mitsuba(context):
  File "C:\Users\dgsd\AppData\Roaming\Blender Foundation\Blender\3.3\scripts\addons\mitsuba-blender\__init__.py", line 60, in try_register_mitsuba
    could_init_mitsuba = init_mitsuba(context)
  File "C:\Users\dgsd\AppData\Roaming\Blender Foundation\Blender\3.3\scripts\addons\mitsuba-blender\__init__.py", line 37, in init_mitsuba
    mitsuba.set_variant('scalar_rgb')
AttributeError: module 'mitsuba' has no attribute 'set_variant'
```

**Environment (please complete the following information):**
 - OS: Windows
 - Blender Version: 3.3
 - Add-on Version: latest

**Comment by ros-dorian** [comment]
Hello,

Mitsuba needs to be installed in Blender's python environment (`blender/3.3/python/bin/python.exe`). Can you try installing Mitsuba there (`python -m pip install mitsuba==3.0.1 --force-reinstall`) ?

In any case, I would suggest waiting for #44 to be merged as it fixes some problems due to dependency installation versions.

Cheers!

**Comment by 240db** [comment]
First I had mitsuba 3.1.1 but it was importing fine on my Ryzen CPU (no GPU, no CUDA, just the integrated graphics)  in Blender's python environment (blender/3.3/python/bin/python.exe). 

But it wouldn't load because only 3.0.1 is supported. So on trying to fix with it by installing mitsuba==3.0.1. I am now unable to import mitsuba 3.0.1 probably because I don't have a CUDA device on this machine? I will try in a different system which has a CUDA card, but could the Blender add-on be compatible with version 3.1.1 as it is successfully importing in Blender's python on systems without CUDA, but this latest mitsuba version is not supported by the latest add-on version.  

Here is my error log when importing 3.0.1 on Blender's python

>>> import mitsuba
jit_cuda_init(): cuInit failed, disabling CUDA backend.
There are two common explanations for this type of failure:

 1. your computer simply does not contain a graphics card that supports CUDA.

 2. your CUDA kernel module and CUDA library are out of sync. Try to see if you
    can run a utility like 'nvida-smi'. If not, a reboot will likely fix this
    issue. Otherwise reinstall your graphics driver.

 The specific error message produced by cuInit was
   "no CUDA-capable device is detected"

Im not sure it's a error or a warning but either way, can't enable Mitsuba Blender add-on with either mitsuba==3.0.1 nor mitsuba==3.1.1 . Tested on Windows Blender 2.93

**Comment by bathal1** [comment]
Closing for inactivity.
